04.ls/ls_command.rb

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+COL_COUNT = 3
+
+def main
+  all_files = find_all_files
+  visible_files = delete_hiddenfile(all_files)
+  display_files(visible_files)
+end
+
+def find_all_files
+  Dir.entries('.').sort
+end
+
+def delete_hiddenfile(files)
+  files.reject { |file| file.start_with?(/\./) }
```

Comment:
```suggestion
  files.reject { |file| file.start_with?(".") }
```

正規表現じゃない方がシンプルかなと思います